CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ Bugfix:
 Enhancement:
 
 - Added `5.1.1.0` to `Get-PasswordPolicyDefault` and `Get-PasswordPolicyConfig` to support VMware Cloud Foundation 5.1.1. [GH-164](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-password-management/pull/164)
+- Added `PasswordRotationTestsuite` with sample testcase for NSX Edge Password Rotation. [GH-185](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-password-management/pull/185)
```

Comment:
```suggestion
```

I think we should move this the Chore section and mention the addition of testcases in general and the PR links.